docs/guests.md

```diff
@@ -43,14 +43,26 @@ bash /mnt/Linux/install.sh -d rhel -m 7
 
 The likeliness for the installation to work correctly will depend on how much those distros differ from their "parent".
 
-## FreeBSD/OpenBSD
+## FreeBSD
 
-On FreeBSD/OpenBSD, the xen drivers are also already part of the kernel.
+FreeBSD is a 30-year-old operating system used widely to run all sorts of systems and has served as the basis for a number of operating systems, including MacOS, pfSense, and FreeNAS.
 
-The `install.sh` script doesn't support those systems, but there are ways to install the management agent anyway.
+The `install.sh` script doesn't support FreeBSD, so there is no need to mount the guest tools ISO. Instead, for FreeBSD guests, you need to install two [ports](https://www.freebsd.org/ports/):
```

Comment:
Why do you need both xe-guest-utilities and xen-guest-tools? What do they do, respectively? Does xen-guest-tools contain the kernel drivers?

Reply:
This is a quirk of how FreeBSD packages things. The [xen-guest-tools](https://www.freshports.org/sysutils/xen-guest-tools/) installs the FreeBSD versions of the xenstore binaries. The [xe-guest-utilities](https://www.freshports.org/sysutils/xe-guest-utilities/) is basically just a single script `xe-daemon` that runs continuously. It periodically calls `xe-update-guest-attrs` to update values. FreeBSD has apparently separated these two into separate packages. One could have the xen guest tools but not have the daemon running.

In reality, one can type the command `pkg install xe-guest-utilities` and the pkg system will figure out that the runtime dependencies include `xen-guest-tools`. It will then prompt the user saying "here's what we're gonna install" just like any interactive `apt` or `rpm` command line would.

Reply:
Ok, then the FreeBSD kernel does contain xen drivers? If that is so, this should be mentioned at the top of the section, as we did for other systems.

Reply:
Right. I've mentioned that. And I've anticipated that my install.sh changes will get accepted, so I rewrote the text to say FreeBSD is supported by install.sh. 😄

Reply:
The updated `install.sh` will probably not reach users before XCP-ng 8.2 so it's a bit early to write that it supports it :)

Reply:
Ok. I adjusted it to just reflect that FreeBSD is not supported yet.